manifest.toml

```diff
@@ -17,12 +17,13 @@ admindoc = "https://docs.moodle.org/400/en/Main_page"
 code = "https://github.com/moodle/moodle"
 
 [integration]
-yunohost = ">= 12.0.9"
+yunohost = ">= 11.2.30"
```

Comment:
@yalh76 Are you gonna revert all my changes?

Reply:
There is no good reason to force 12.0.9, as the package still works on bullseye or in that case, see to remove bullseye tests and make YunoHost 12.0.9 available for trixie

Because for now, forcing 12.0.9 you only make bullseye and trixie tests failed

Reply:
and I'm not trying to revert the changes you made, just trying to solve issues on not-working package or update